includes/functions/llms-functions-l10n.php

```diff
@@ -91,7 +89,9 @@ function llms_load_textdomain( $domain, $plugin_dir = null, $language_dir = null
 	$plugin_dir   = $plugin_dir ? $plugin_dir : LLMS_PLUGIN_DIR;
 	$language_dir = $language_dir ? $language_dir : 'languages';
 
-	unload_textdomain( $domain );
+	if ( 'lifterlms' !== $domain ) {
```

Comment:
We may want to just remove this completely even for add-ons?

Reply:
Yes. Let's remove it completely. Then test and if we don't have the issue in the latest versions of WP, Loco, and Lifter, merge it.